platform-tools-sdk/cargo-build-sbf/src/main.rs

```diff
@@ -557,6 +559,14 @@ fn main() {
              decrease program size and CU consumption. The default option is LTO disabled, as one \
              may get mixed results with it.",
         ))
+        .arg(
+            Arg::new("patch_nix")
+                .long("patch-binaries-for-nix")
+                .takes_value(true)
```

Comment:
nit: in our CLIs, we typically have flags just enable a feature without also specifying `true` or `false`, so you just need to specify `--patch-binaries-for-nix`, and not `--patch-binaries-for-nix true`.

So this shouldn't take a value and just check if it's present.